### PR TITLE
Refactoring of IconBrandColor - nps.today certified connector

### PR DESCRIPTION
**Pull request by HazemSaeid**
Refactored IconBrandColor
added .venv to .gitignore

---
When submitting a connector, please check the following conditions for your PR.

- [ X] I attest that the connector works and I verified by deploying and testing all the operations.
- [ X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a logo.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -360,4 +360,5 @@ build/
 dist/
```
Comment (sriyen-msft) [inline .gitignore:360]: @HazemSaeid Please remove this file from this PR

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (HazemSaeid) [reply .gitignore]: Hi, sorry for the late answer. The .gitignore file is now deleted.

**Comment by HazemSaeid** [comment]
Hi @sriyen-msft - anything new?

**Comment by HazemSaeid** [comment]
> Hi @sriyen-msft - anything new?

Just saw your PR at my repo. File has been reverted. Sorry for the misunderstanding, the "Outdated" tag confused me.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**